edtrio-client/src/App.tsx

```diff
@@ -26,10 +26,24 @@ export default function App() {
         >
           <EditorWrapper
             // TODO: get this from the props (when included in schul-cloud client)
-            documentId={"cjpcys08y002607989geb9ttk"}
+            documentId={
+              getQueryVariable("documentId") || "cjqm7lirq00sh0740clb48905"
+            }
           />
         </EditorStateProvider>
       </LastSavedProvider>
     </ThemeProvider>
   );
 }
+
+const getQueryVariable = (variable: string) => {
```

Comment:
You're absolutely right, I would also prefer to use a cleaner way. However, due to our target audience with schools, I fear we should not drop support with IE based on that little thing. As soon as the integration with the Schul-Cloud is done, I hope that this might not be required any more.